demo/atom/input/playground

```diff
@@ -215,6 +216,21 @@ return (
         rightAddon="@schibsted.com"
       />
     </div>
+    <div style={field}>
+      <AtomInput
+        name="second"
+        placeholder="Medium Input"
+        leftIcon={LeftAddonComponent}
+      />
+    </div>
+    <div style={field}>
+      <AtomInput
+        name="second"
+        placeholder="Medium Input"
+        leftIcon={LeftAddonComponent}
```

Comment:
there should be no left addon if a left icon is set, from the paper ```If it is combined with an icon, the addon must be placed to the right side, because the icon must be placed at the left side.```

Reply:
but only remove the example